exporter/exporterhelper/logs.go

```diff
@@ -85,18 +85,24 @@ func NewLogsExporter(
 
 	bs := fromOptions(options...)
 	be := newBaseExporter(cfg, logger, bs)
+	obsrep := obsreport.NewExporter(obsreport.ExporterSettings{
+		Level:      configtelemetry.GetMetricsLevelFlagValue(),
+		ExporterID: cfg.ID(),
+	})
```

Comment:
Probably can be configured in the `newBaseExporter?`

Reply:
sg, moved